flow/scripts/run_all.tcl

```diff
@@ -58,6 +60,8 @@ if {[info exists ::env(USE_FILL)]} {
 }
 
 # Finishing
+# make sure to save final files
+set save_checkpoint 1
 source -verbose $::env(SCRIPTS_DIR)/final_report.tcl
 source -verbose $::env(SCRIPTS_DIR)/klayout.tcl
```

Comment:
Don't we always want to save the results in these scripts?  Why make them conditional on save_checkpoint at all?

Reply:
Fixed on 9066de9